Description: Fix for g++ 13 standards change
Author: Alastair McKinstry <mckinstry@debian.org>
Last-Updated: 2023-06-15
Forwarded: no
Bug-Origin: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1037796


--- a/src/odc/api/StridedData.h
+++ b/src/odc/api/StridedData.h
@@ -15,6 +15,7 @@
 #ifndef odc_api_StridedData_H
 #define odc_api_StridedData_H
 
+#include <cstdint>
 #include <cstddef>
 #include <string.h>
 #include <algorithm>
